README.md

```diff
@@ -1,38 +1,51 @@
-Role Name
-=========
+SSL Certificates
+================
 
-A brief description of the role goes here.
+Manage SSL certificates for web-servers.
 
-Requirements
-------------
+Optionally generate self-signed SSL certificates for internal testing.
 
-Any pre-requisites that may not be covered by Ansible itself or the role should be mentioned here. For instance, if the role uses the EC2 module, it may be a good idea to mention in this section that the boto package is required.
 
 Role Variables
 --------------
 
-A description of the settable variables for this role should go here, including any variables that are in defaults/main.yml, vars/main.yml, and any variables that can/should be set via parameters to the role. Any variables that are read from other roles and/or the global scope (ie. hostvars, group vars, etc.) should be mentioned here as well.
+Defaults: `defaults/main.yml`
 
-Dependencies
-------------
+Optional variables:
+- `ssl_certificate_path`: Server path to SSL certificate
```

Comment:
How do you see these variable names changing with gh-3?

Reply:
Probably something like:
- `ssl_certificate_path`: single certificate
- `ssl_certificate_intermediate_path`: intermediate bundle
- `ssl_certificate_bundled_path`: what nginx uses ... so could change `ssl_certificate_path` to this now to avoid a future breaking change?

Reply:
If you have a feeling for what the next round will need, then yeah, avoiding breakage sounds great.

Reply:
Probably worth anticipating indeed, the other issue with `ssl_certificate_path` is that is can be confused with the path to the SSL folder e.g. `/etc/ssl/` or `/etc/nginx/ssl`.

Following what has been proposed here, I like the idea of prefixing the names with the type of certificate. Is the difference by file format or by type of certificate?

Reply:
Have a look at https://github.com/openmicroscopy/management_tools/blob/master/sa/ssl/https-certificates.md#nginx-godaddy-certificate-installation